src/fiskaltrust.ifPOS/v1/it/Models/States.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Runtime.Serialization;
+
+namespace fiskaltrust.ifPOS.v1.it
+{
+    /// <summary>
+    ///  rtMainStatus indicates the RT main status
+    /// </summary>
+    [DataContract]
+    public enum MainState
```

Comment:
The existence of these states make me think that there probably are some "lifecycle" methods, e.g. to initialize/decommission the printer. Let's fine out if these can be set programmatically 😁